pom.xml

```diff
@@ -194,6 +199,16 @@
                             </execution>
                         </executions>
                     </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>coveralls</id>
+            <activation>
+                <file><exists>target/site/cobertura/coverage.xml</exists></file>
```

Comment:
Now cobertura always works, coveralls - only if there is target/site/cobertura/coverage.xml.